src/main/java/edu/harvard/iq/dataverse/util/FileUtil.java

```diff
@@ -1834,7 +1834,8 @@ public static boolean isPackageFile(DataFile dataFile) {
     
     public static S3AccessIO getS3AccessForDirectUpload(Dataset dataset) {
     	String driverId = dataset.getEffectiveStorageDriverId();
-    	boolean directEnabled = Boolean.getBoolean("dataverse.files." + driverId + ".upload-redirect");
+    	boolean directEnabled = Boolean.parseBoolean(System.getProperty("dataverse.files." + driverId + ".upload-redirect")) ||
+            Boolean.parseBoolean(System.getProperty("dataverse.files." + driverId + ".upload-out-of-band"));
```

Comment:
Should this be here? Is this path used for out-of-band uploads? If one has upload-out-of-band true and upload-redirect false for an S3 store, I assume we'd want calls to generate the S3 direct upload URLs to fail.? (and I guess I'm assuming that this is a valid config.) With this here, I don't think those calls would fail. 

I can see that it would be somewhat useful to get a valid storageidentifier from Dataverse at the start of the out-of-band case - so that the external tool/manual process doesn't have to invent a valid one. Perhaps that's a useful call to have, separate from one that also generates the upload URLs for S3? (If so - I may be able to add that in the Globus work (where Globus is similar to an out-of-band tool) rather than delaying this PR.)

Reply:
I went through the code, it is not used in out-of-band uploads. I have reverted that part.